spec/support/feature_helpers/agent.rb

```diff
@@ -0,0 +1,12 @@
+module FeatureHelpers
+  module AgentHelper
+
+    def an_agent_exists
```

Comment:
This is because when I tried to select certain agent from the dropdown I usually got a not found error as there was empty data pulled from the test_db. So Cristian had recommended to do a helper in which an agent was created with a username. This enabled me to select an agent an move along in the happy trail of testing.